library/oneview_version_facts.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: oneview_version
+short_description: Returns the range of possible API versions supported by the appliance.
+description:
+    - Provides an interface to return the range of possible API versions supported by the appliance.
+version_added: "2.3"
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView >= 3.2.0"
+author: "Priyanka Sood (@soodpr)"
+options:
+    state:
+        description:
+            - Indicates the desired state for the Appliance current and minimum version.
+              C(present) will ensure data properties are compliant with OneView.
+        choices: ['present']
+    data:
```

Comment:
We don't usually have a `data` for facts, and this resource also does not have a use for that data.